CondTools/Geometry/test/calogeometrywriter.py

```diff
@@ -2,7 +2,8 @@
 
 process = cms.Process("CaloGeometryWriter")
 process.load('CondCore.DBCommon.CondDBCommon_cfi')
-process.load('Configuration.StandardSequences.GeometryExtended_cff')
+process.load('Configuration.Geometry.GeometryExtended_cff')
+process.load('Configuration.Geometry.GeometryReco_cff')
```

Comment:
Taken your advice. But StandardSequences.GeometryExtended_cff does not have initiator for recnumbering record which is needed and when I ran without Geometry.GeometryReco_cff it did not work. Hcal geometry records for run 1 and run 2 are fine - the error was only for SLHC version which was in the 62X version and was never tested.
________________________________
From: Ianna Osborne [notifications@github.com]
Sent: 29 September 2015 07:43
To: cms-sw/cmssw
Cc: Sunanda Banerjee
Subject: Re: [cmssw] bsunada:Run2-hcx32 Bug fix for HcalTopology (#11538)


In CondTools/Geometry/test/calogeometrywriter.py<https://github.com/cms-sw/cmssw/pull/11538#discussion_r40636326>:

> @@ -2,7 +2,8 @@
>
>  process = cms.Process("CaloGeometryWriter")
>  process.load('CondCore.DBCommon.CondDBCommon_cfi')
> -process.load('Configuration.StandardSequences.GeometryExtended_cff')
> +process.load('Configuration.Geometry.GeometryExtended_cff')
> +process.load('Configuration.Geometry.GeometryReco_cff')


@bsunanda<https://github.com/bsunanda> - please, remove this line. All relevant producers are include through the next one with appropriate parameters. Besides, if this file is not modified, the PR will be merged faster.

Another question: does it mean Hcal geometry records in DB are wrong and need to be recreated? Thanks

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/11538/files#r40636326>.